html/changelog.html

```diff
@@ -51,6 +51,32 @@
 		
 		-->
 		<div class="commit sansserif">
+			<h2 class="date">04 September 2020</h2>
+			<h3 class="author">Fikou updated:</h3>
+			<ul class="changes bgimages16">
+				<li class="tweak">spy glasses now use an action button instead of a verb, so they are less cryptic and someone will know how to use them</li>
+				<li class="balance">spy glasses are now sunglasses, removes the sunglasses from det's office so you can just take the spy ones from your locker</li>
+			</ul>
+			<h3 class="author">Ghilker updated:</h3>
+			<ul class="changes bgimages16">
+				<li class="bugfix">Icebox pipe layering error</li>
+			</ul>
+			<h3 class="author">SpaceManiac updated:</h3>
+			<ul class="changes bgimages16">
+				<li class="bugfix">Zipper noises are no longer played on the lobby screen when a character joins with a random-colored jumpsuit.</li>
+			</ul>
+			<h3 class="author">itseasytosee updated:</h3>
+			<ul class="changes bgimages16">
+				<li class="bugfix">supermatter tongs holding supermatter shards now have working in hand sprites</li>
+			</ul>
+			<h3 class="author">nightred updated:</h3>
+			<ul class="changes bgimages16">
+				<li class="tweak">The stomach is now needed to eat food or drink</li>
+				<li class="tweak">Vomiting now empties the stomach only,</li>
+				<li class="rscadd">New surgery to filter the blood of reagents and the tool that is needed</li>
+				<li class="tweak">Stomach pump surgery is now do able on the living</li>
+			</ul>
+
```

Comment:
Still a thing. Disable your github actions and revert this commit.

Reply:
I think it's finally dead. If it comes back the next time I merge master I don't know what I'll do.

Reply:
It stayed gone. I think it's done.